### DISCUSSION TITLE
Not able to customize tailwind grids?

### DISCUSSION THREAD

**Discussion by nitesh5692** [tailwindlabs/tailwindcss#3407] [Help]

**Comment by simonswiss** [comment]
Hey! 👋

You can [customize the number of rows/columns](https://tailwindcss.com/docs/grid-template-columns#customizing) as well as the [gap property](https://tailwindcss.com/docs/gap#customizing), that's about it.

Did you have some more specific grid customisation in mind?

**Reply by Greensboroo** [reply]
The plugins section allows you to register plugins with Tailwind that can be used to  Note that any of your own custom utilities will not automatically be marked as !important by  gridAutoRows, The grid-auto-rows utilities like auto-rows-min.

[Tellpizzahut Survey](https://www.tellpizzahut.one/)
